src/dialogsearch.cpp

```diff
@@ -65,7 +65,32 @@ void DialogSearch::on_button_search_clicked()
     sl << "Map" << "Event" << "Event Page" << "Sourceline" << "Action";
     ui->list_result->setHorizontalHeaderLabels(sl);
 
-    std::function<std::vector<command_info>(const RPG::Map&)> map_searcher;
+    std::function<bool(const RPG::EventCommand&)> search_predicate;
+    auto map_searcher = [&search_predicate](const RPG::Map& map) {
+        std::vector<command_info> res;
+        for (auto& e : map.events)
+            for (auto& p : e.pages)
+                for (size_t line = 0; line < p.event_commands.size(); line++)
+                {
+                    auto& com = p.event_commands[line];
+                    if (search_predicate(com))
+                        res.emplace_back(map.ID, e.ID, p.ID, line, com);
```

Comment:
Today it was the first time, that I saw the tooltip in the event-editor and that one is also 0-based. Do we want to change this or leave it? (Also the tooltip could appear a little bit faster)